### ISSUE TITLE
After webview2 is loaded, it shows that there is an issue loading this source. Please refresh the source

### ISSUE THREAD

**Issue by xiakan** [MicrosoftEdge/WebView2Feedback#4405]
### What happened?

After webview2 is loaded, it shows that there is an issue loading this source. Please refresh the source and click on the refresh source link. The upper part displays the refresh source, while the lower part displays the webpage.

### Importance

Important. My app's user experience is significantly compromised.

### Runtime Channel

Stable release (WebView2 Runtime)

### SDK Version

1.0.1661.34

### Framework

WinUI2/UWP

### Operating System

Windows 11

### Repro steps

probabilistic

### Repros in Edge Browser

No, issue does not reproduce in the corresponding Edge version

### Regression

No, this never worked

**Comment by xiakan** [comment]
![Weixin Image_20240118180233](https://github.com/MicrosoftEdge/WebView2Feedback/assets/33214137/1b2c5dcd-32b9-4119-a3d5-28e58ebe6dd3)
![Weixin Image_20240118180034](https://github.com/MicrosoftEdge/WebView2Feedback/assets/33214137/d09693f1-7548-4678-a607-c615479b7a17)

**Comment by xiakan** [comment]
Yes

**Comment by xiakan** [comment]
This problem has always existed

**Comment by victorhuangwq** [comment]
> Repros in Edge Browser
> No, issue does not reproduce in the corresponding Edge version

And this problem does not occur in the Edge Browser?

**Comment by xiakan** [comment]
No, there's no problem opening it with a link. This issue may occur when loading in webview2

**Comment by xiakan** [comment]
Can anyone answer this question?
